Assets/Scripts/ShootEnemies.cs

```diff
@@ -17,18 +17,21 @@ public class ShootEnemies : MonoBehaviour
     Queue<GameObject> enemiesInRange;
     float lastShot;
     Transform cannonTransform;
+    CapsuleCollider collider;
 
     // Start is called before the first frame update
     void Start()
```

Comment:
What is the purpose of this?

Reply:
this is so we can resize the trigger to match the new radius after you upgrade the radius

Reply:
ohhh i see